src/components/Header/Header.tsx

```diff
@@ -11,20 +11,18 @@ import { useMediaQuery } from '@mui/material';
 import MenuIcon from '@mui/icons-material/Menu';
 import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
 import ChevronRightIcon from '@mui/icons-material/ChevronRight';
-import Button from '../Button';
+import Logo from '../../img/Logo.png';
```

Comment:
This needs to be a local file (in same folder), otherwise rollup will complain. 

Consider using absolute path, ie: `/img/Logo.png`

Reply:
You mean having Logo inside Header folder? or inside Components folder?

Reply:
Review this Stackoverflow for clearer answer: https://stackoverflow.com/questions/57061414/rollup-changes-import-of-absolute-path-to-relative-path-while-bundling

Reply:
Do you know if there is a way to test it? Because I've added it same way as from the link you provided and I have no idea if it will work

Reply:
The way I would test it is:

`npm run build`

Then you can try importing the package using the README guide. This is good for now. I will close this comment